### PR TITLE
Add Paperspace.app latest

### PR DESCRIPTION
**Pull request by sirn**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
I don’t see the app anywhere on the website. Where is it?

**Comment by sirn** [comment]
tIt's a little bit hard to find without an account, but it's located at https://paperspace.io/download 

The only non-behind-account I could found is:

1. Official site
2. "Help Desk" link at the bottom
3. "Help Desk" 
4. Technical
5. Where do I download the native app

File: Casks/paperspace.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'paperspace' do
+  version :latest
+  sha256 :no_check
+
+  # amazonaws.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/paperspace.rb]: This comment is outdated. It should be `# s3-us-west-1.amazonaws.com/ps-receiver was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 600a52622af474f11a49091afbfbcf9400642dcb. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**